staging/src/k8s.io/apiserver/pkg/cel/mutation/env_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mutation
+
+import (
+	"testing"
+
+	"github.com/google/cel-go/cel"
+
+	"k8s.io/apimachinery/pkg/util/version"
+	"k8s.io/apiserver/pkg/cel/environment"
+)
+
+func mustCreateEnv(t testing.TB, envOptions ...cel.EnvOption) *cel.Env {
+	envSet, err := environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()).
+		Extend(environment.VersionedOptions{
+			IntroducedVersion: version.MajorMinor(1, 30),
```

Comment:
Just wanna mention that we introduced optional in 1.28 so it should be ready to use by the time mutation is introduced :)

Reply:
Good hint! Note that, however, in the test environment we still need to set it up manually.